charts/argo-applicationset/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: argocd-applicationset
 description: A Helm chart for installing ArgoCD ApplicationSet
 type: application
-version: 0.1.0
-appVersion: "v0.1.0-prerelease"
+version: 0.2.0
```

Comment:
@maruina What if we go with `version: 0.1.1`, so that the product version and appVersion are equivalent (for the time being). Since Helm Chart 0.1.0 was a pre-release version, I think backwards/forward compatibility requirements are less strict versus a normal release. This then allows us to track the product version in the future (until such time as we need to diverge again).

Reply:
Done